docs/content/guides/developer/app-examples/plinko.mdx

```diff
@@ -0,0 +1,772 @@
+        // Only the house address can update the min stake.
+        assert!(tx_context::sender(ctx) == house(house_data), ECallerNotHouse);
+
+        house_data.min_stake = min_stake;
+    }
+
+    /// Returns a mutable reference to the balance of the house.
+    public(friend) fun borrow_balance_mut(house_data: &mut HouseData): &mut Balance<SUI> {
+        &mut house_data.balance
+    }
+
+    /// Returns a mutable reference to the fees of the house.
+    public(friend) fun borrow_fees_mut(house_data: &mut HouseData): &mut Balance<SUI> {
+        &mut house_data.fees
+    }
+
+    /// Returns a mutable reference to the house id.
+    public(friend) fun borrow_mut(house_data: &mut HouseData): &mut UID {
+        &mut house_data.id
+    }
+
+    // === Public-View Functions ===
+
+    /// Returns a reference to the house id.
+    public(friend) fun borrow(house_data: &HouseData): &UID {
+        &house_data.id
+    }
+
+    /// Returns the balance of the house.
+    public fun balance(house_data: &HouseData): u64 {
+        balance::value(&house_data.balance)
+    }
+
+    /// Returns the address of the house.
+    public fun house(house_data: &HouseData): address {
+        house_data.house
+    }
+
+    /// Returns the public key of the house.
+    public fun public_key(house_data: &HouseData): vector<u8> {
+        house_data.public_key
+    }
+
+    /// Returns the max stake of the house.
+    public fun max_stake(house_data: &HouseData): u64 {
+        house_data.max_stake
+    }
+
+    /// Returns the min stake of the house.
+    public fun min_stake(house_data: &HouseData): u64 {
+        house_data.min_stake
+    }
+
+    /// Returns the fees of the house.
+    public fun fees(house_data: &HouseData): u64 {
+        balance::value(&house_data.fees)
+    }
+
+    /// Returns the base fee.
+    public fun base_fee_in_bp(house_data: &HouseData): u16 {
+        house_data.base_fee_in_bp
+    }
+
+    /// Returns the multiplier vector
+    public fun multiplier(house_data: &HouseData): vector<u64> {
+        house_data.multiplier
+    }
+
+    // === Test Functions ===
+
+    #[test_only]
+    public fun init_for_testing(ctx: &mut TxContext) {
+        init(HOUSE_DATA {}, ctx);
+    }
+}
+```
+
+#### Error codes
+
+The module defines specific error codes to handle exceptional scenarios:
+
+- `ECallerNotHouse`: Ensures that only the house (game operator) can perform certain operations.
+- `EInsufficientBalance`: Indicates an insufficient balance for operations requiring a minimum fund level.
+
+#### Structures
+
+- `HouseData`: A key configuration object storing the house's operational parameters, including its balance, public key, stake limits, accumulated fees, base fee rate, and multiplier settings for game outcomes.
+- `HouseCap`: A unique capability indicating the authority to initialize the house data.
+- `HOUSE_DATA`: A marker for initializing house data, used once during the setup phase.
```

Comment:
- `HOUSE_DATA`: A one time witness for initializing house data, used once during the setup phase.